pyvista/core/filters/data_set.py

```diff
@@ -3261,6 +3266,11 @@ def point_data_to_cell_data(self, pass_point_data=False, progress_bar=False):
         pass_point_data : bool, default: False
             If enabled, pass the input point data through to the output.
 
+        categorical : bool, default: False
+            Control whether the source point data is to be treated as
+            categorical. If the data is categorical, histograming will be used
+            to assign the cell data.
+
```

Comment:
In response to this thread https://github.com/pyvista/pyvista/pull/6407#discussion_r1691956369, how about adding this explanation? Any thoughts on this change @nicolascedilnik ?

```suggestion
        categorical : bool, default: False
            Control whether the source point data is to be treated as
            categorical. If ``True``,  histograming is used to assign the 
            cell data. Specifically, a histogram is populated for each cell
            from the scalar values at each point, and the bin with the most
            elements is selected. In case of a tie, the smaller value is selected.
            
            .. note::
             
                If the point data is continuous, values that are almost equal (within 
                ``1e-6``) are merged into a single bin. Otherwise, for discrete data
                the number of bins equals the number of unique values.

```

Reply:
It's fine with me, of course!

Reply:
@user27182 Could you commit this proposal before we forget?